### PR TITLE
Fixes #22071 - Upgrade patternfly-sass to v3.32.1

### PR DESCRIPTION
**Pull request by sharvit**
Upgrade patternfly-sass to v3.32.1

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/notifications/notifications.scss [hunk 1]
```diff
@@ -40,21 +36,6 @@ $color-pf-blue-400: #0088ce;
...
-      position: absolute;
-
-      &:hover,
-      &:focus {
-        color: $color-pf-blue-400;
-        text-decoration: none;
-      }
-    }
-
     .drawer-pf-close {
```
Comment (ohadlevy) [inline webpack/assets/javascripts/react_app/components/notifications/notifications.scss:39]: I thought that most / all of these css can go away?

Comment (sharvit) [reply webpack/assets/javascripts/react_app/components/notifications/notifications.scss:39]: Yes, I am working on it on a different pr. It's need a bigger refactoring.

**Comment by ohadlevy** [comment]
@waldenraines can you review please?

**Review by waldenraines — APPROVED** [review]
LGTM

**Review by ohadlevy — APPROVED** [review]

**Pull request merged**